.clang-format

```diff
@@ -0,0 +1,33 @@
+BasedOnStyle:  WebKit
+---
+Language:        Cpp
+AlignConsecutiveMacros: true
+AlignOperands:   Align
+AlignTrailingComments: true
+AllowAllParametersOfDeclarationOnNextLine: true
+BinPackParameters: false
+ColumnLimit:     120
+IncludeCategories:
+  - Regex:           '^<openssl/'
+    Priority:        2
+    SortPriority:    0
+  - Regex:           '^"(llvm|llvm-c|clang|clang-c)/'
+    Priority:        3
+    SortPriority:    0
+  - Regex:           '^(<|"(gtest|gmock|isl|json)/)'
+    Priority:        4
+    SortPriority:    0
+  - Regex:           '.*'
+    Priority:        1
+    SortPriority:    0
+MaxEmptyLinesToKeep: 3
+WhitespaceSensitiveMacros:
+  - STRINGIZE
+  - PP_STRINGIZE
+  - BOOST_PP_STRINGIZE
+  - TO_STRING
+  - TO_STRING_0
+  - CONCAT2
+  - CONCAT2_INTERNAL
+...
```

Comment:
what's the ellipsis here? is that a YAML thing?

Reply:
Yeah, It looks to be a YAML thing that I don't fully understand. This is a modified YAML generated by clang-format.